PizzaDAO/pizza-oven-py
# Feature/invoke natron

connects the natron render pipeline ot the pai and puts it in docker

## To Test
1. follow the readme to stand up the docker containers locally
2. you can also run `make start` and run just the python app server without docker, if you want
2. download the postman files and run through the requests (get recipe, get kitchen order, bake).

expected: the bake request takes about a minute and then returns a successful result

1. install natron
2. run `make test-natron`
2. wait for it to finish

expected: natron renders a bunch of stuff. check the output folder to verify there are things tehre and that they all render (open the images).  note there are a couple errors as of right now but they can ber ignored.

### chainlink

1. if you are feeling really fancy, you can follow the instructions in the readme and log into the chainlink node on localhost, set everything up and issue a request from the chainlink node
